components/Feature/GoalSummary/index.js

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+import { SummaryList } from 'components/Form';
+
+const GoalSummary = ({ goal }) => {
+  return (
+    <SummaryList
+      name="goal-summary"
+      listObject={{
+        Goal: goal.text,
+        'Target review date': moment(goal.targetReviewDate)
+          .utc()
+          .format('DD/MM/YYYY')
```

Comment:
We used `Intl` for formatting dates, but that's cool we can merge and then pick one afterwards.